ui/pages/settings/flask/snaps-list-tab/snap-list-tab.component.js

```diff
@@ -0,0 +1,110 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import SnapSettingsCard from '../../../../components/app/flask/snap-settings-card';
+import {
+  removeSnap,
+  removeSnapPermissionsFromSubjects,
+} from '../../../../store/actions';
+import { useI18nContext } from '../../../../hooks/useI18nContext';
+import Typography from '../../../../components/ui/typography/typography';
+import {
+  TYPOGRAPHY,
+  COLORS,
+  FLEX_DIRECTION,
+} from '../../../../helpers/constants/design-system';
+import Box from '../../../../components/ui/box';
+import ViewSnap from './view-snap';
+
+const propTypes = {
+  snaps: PropTypes.object.isRequired,
+  viewingSnap: PropTypes.bool,
+  currentSnap: PropTypes.object,
+  onClick: PropTypes.func.isRequired,
+  onRemove: PropTypes.func.isRequired,
+  onToggle: PropTypes.func.isRequired,
+  dispatch: PropTypes.func.isRequired,
+};
+
+const SnapListTab = ({
+  snaps,
+  viewingSnap,
+  currentSnap,
+  onClick,
+  onRemove,
+  onToggle,
+  dispatch,
+}) => {
+  const t = useI18nContext();
+  if (viewingSnap && currentSnap) {
+    return (
+      <ViewSnap
+        snap={currentSnap}
+        onToggle={(event) => onToggle(event, currentSnap)}
+        onRemove={async (event) => {
+          await dispatch(removeSnap(currentSnap.id));
+          await dispatch(
+            removeSnapPermissionsFromSubjects(currentSnap.permissionName),
+          );
+          onRemove(event, currentSnap);
+        }}
+      />
+    );
+  }
+  return (
+    <>
+      {Object.entries(snaps).length ? (
+        <div className="snap-settings-page__body">
+          <Box display="flex" flexDirection={FLEX_DIRECTION.COLUMN}>
+            <Typography variant={TYPOGRAPHY.H5} marginBottom={2}>
+              {t('expandExperience')}
+            </Typography>
+            <Typography
+              variant={TYPOGRAPHY.H6}
+              color={COLORS.UI4}
+              marginBottom={2}
+            >
+              {t('manageSnaps')}
+            </Typography>
+          </Box>
+          <div className="snaps-list-wrapper">
+            {Object.entries(snaps).map(([key, snap]) => {
+              return (
+                <SnapSettingsCard
+                  className="snap-settings-card"
+                  isEnabled={snap.enabled}
+                  dateAdded={new Date().toDateString()}
```

Comment:
This always shows the current date. That's not right. Also this format isn't ideal because it doesn't match the users current locale.

Better to remove the date if we don't know when the snap was added. If we to do know, we should format the date with the `Intl.DateTimeFormat` API, which respects the user's current locale.

Reply:
It looks like there is no install date stored in the snaps controller right now, so we can omit this for now.